common/src/main/java/org/apache/sedona/common/Functions.java

```diff
@@ -80,6 +80,123 @@ public static double area(Geometry geometry) {
     return geometry.getArea();
   }
 
+  public static Geometry anchor(Geometry geometry) {
+    return anchor(geometry, 2, 0.2);
```

Comment:
isntead of a step size argument, lets do a grid resolution kind of argument. this makes the runtime and the search quality scale invariant.

Reply:
val stepSize = width / gridResolution

something like this. not sure if we need to handle width and length seperately.

Reply:
@james-willis By width you mean width of the grid/tile? If so, how would one access the width of the grid?

Reply:
width of the extent of the geometry

Reply:
How about diagonal of the MBR as a measure of extent?